quinn-proto/src/connection/streams/state.rs

```diff
@@ -797,24 +832,26 @@ impl StreamsState {
         expanded
     }
 
+    pub(super) fn max_send_data(&self, id: &StreamId) -> VarInt {
```

Comment:
It would be nice to squash all the commits and then extract this as a separate prefix commit.